{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/gtmappauth\/authsession\/error\/errordomain-64zxv"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession\/Error\/errorDomain-64zxv","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"CustomNSError.errorDomain"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"errorDomain","roleHeading":"Type Property","role":"symbol","symbolKind":"property","externalID":"s:10GTMAppAuth0B7SessionC5ErrorO11errorDomainSSvpZ","modules":[{"name":"GTMAppAuth"}]},"hierarchy":{"paths":[["doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth","doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession","doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession\/Error"]]},"references":{"doc://GTMAppAuth/documentation/GTMAppAuth/AuthSession/Error/errorDomain-64zxv":{"role":"symbol","title":"errorDomain","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDomain"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession\/Error\/errorDomain-64zxv","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth\/authsession\/error\/errordomain-64zxv"},"doc://GTMAppAuth/documentation/GTMAppAuth/AuthSession":{"role":"symbol","title":"AuthSession","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"AuthSession"}],"abstract":[{"type":"text","text":"An implementation of the "},{"type":"codeVoice","code":"GTMFetcherAuthorizationProtocol"},{"type":"text","text":" protocol for the AppAuth library."}],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AuthSession"}],"url":"\/documentation\/gtmappauth\/authsession"},"doc://GTMAppAuth/documentation/GTMAppAuth/AuthSession/Error":{"role":"symbol","title":"AuthSession.Error","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Error"}],"abstract":[{"type":"text","text":"Errors that may arise while authorizing a request."}],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth\/AuthSession\/Error","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Error"}],"url":"\/documentation\/gtmappauth\/authsession\/error"},"doc://GTMAppAuth/documentation/GTMAppAuth":{"role":"collection","title":"GTMAppAuth","abstract":[],"identifier":"doc:\/\/GTMAppAuth\/documentation\/GTMAppAuth","kind":"symbol","type":"topic","url":"\/documentation\/gtmappauth"}},"variantOverrides":[{"patch":[{"op":"replace","path":"\/references\/doc:~1~1GTMAppAuth~1documentation~1GTMAppAuth~1AuthSession\/title","value":"GTMAuthSession"},{"op":"replace","path":"\/references\/doc:~1~1GTMAppAuth~1documentation~1GTMAppAuth~1AuthSession\/fragments","value":[{"kind":"identifier","text":"GTMAuthSession"}]}],"traits":[{"interfaceLanguage":"occ"}]}]}